### ISSUE TITLE
Only implemented softmax 

### ISSUE THREAD

**Issue by tudou330121** [jxgu1016/MNIST_center_loss.pytorch#13]
loss = nllloss(pred, target) + loss_weight * centerloss(target, ip1)
the loss is centerloss and softmaxloss，the result is correct
when i remove the centerloss, loss = nllloss(pred, target),
the result is incorrect.
![image](https://user-images.githubusercontent.com/30098601/50506637-da0a2600-0ab4-11e9-954a-70fb4f7f4e8e.png)

**Comment by jxgu1016** [comment]
Try not to set ``` plt.xlim``` and ``` plt.x=ylim``` to fixed values.

**Comment by jxgu1016** [comment]
BTW, setting the ```loss_weight``` to 0 is preferred.

**Comment by tudou330121** [comment]
What is the difference between setting w to 0 and removing centerloss directly?

**Comment by tudou330121** [comment]
Thank you for your answer，it is correct to  set plt.xlim and plt.ylimto a non-fixed value.
